app/api/app.py

```diff
@@ -89,9 +66,65 @@ def current_user(is_user_expected: bool = True) -> Optional[User]:
     current = g.get("current_user")
     if is_user_expected and current is None:
         logger.error("No current user found for request")
-        raise Unauthorized
+        # raise Unauthorized
     return current
 
 
+def configure_app(app: APIFlask) -> None:
+    # Modify the response schema to instead use the format of our Response class
+    # which adds additional details to the object.
+    app.config["BASE_RESPONSE_SCHEMA"] = marshmallow_dataclass.class_schema(Response)
+
+    # Set a few values for the Swagger endpoint
+    app.config["OPENAPI_VERSION"] = "3.0.3"
+
+    # Set various general OpenAPI config values
+    app.info = {
+        "title": "Template Application Flask",
+        "description": "Template API for a Flask Application",
+        "contact": {
+            "name": "Nava PBC Engineering",
+            "url": "https://www.navapbc.com",
+            "email": "engineering@navapbc.com",
+        },
+    }
```

Comment:
what's the difference between `app.config["INFO"] = `and `app.info = `? if there's no difference could we use `app.config["INFO"]` to be consistent with the previous configuration settings, or conversely change the other ones to use the attribute method like `app.open_api_version`?

if one of them gives us better type safety then maybe we can use that one (e.g. if the attributes are defined at compile-time then that might could be checked by mypy)

Reply:
what do you think about leaving in placeholder values like
```suggestion
    app.info = {
        "title": "<APPLICATION_NAME>",
        "description": "<APPLICATION_DESCRIPTION>",
    }
```